dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -138,6 +138,15 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
         return hasSetTaskEntrypoint;
     }
 
+    inline bool IsElevated()
```

Comment:
Was this copied over from somewhere else in the SDK? If so, might make sense to move this to the common package.

Reply:
Did not know Security::IntegrityLevel::IsElevated was provided by Security.IntegrityLevel.h. Switching to that helper API instead.